parcels/tools/exampledata_utils.py

```diff
@@ -72,6 +72,7 @@
         "field_0065557.nc",
     ],
     "WOA_data": [f"woa18_decav_t{m:02d}_04.nc" for m in range(1, 13)],
+    "CROCOidealized_data": ["CROCO_idealized.nc"],
```

Comment:
Should we remove this dataset from the 'test_data' folder and just rely on the data fetching for both the notebook and the test cases?

Reply:
Good idea; done now